magento/adobe-stock-integration
# fix Used in entity is removed after and Image Preview has been Licensed

### Description (*)


### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1721: Used in entity is removed after and Image Preview has been Licensed
2. ...

### Manual testing scenarios (*)
1. Go to **Content - Media Gallery**
2. Click **Search Adobe Stock** and Save an image Preview
3. Create a new Page and add the previously saved _Unlicensed_ image to its Content (also reproducible on Products, Blocks and Categories)
4. Go to **Content - Media Galery** and select the previously used _Unlicensed_ image
5. **View details** and assert that the image is Used in Page
![used in page](https://user-images.githubusercontent.com/45624059/89304700-6cea4980-d676-11ea-8448-d09b0c71836d.png)
6. Close Image Details
7. Click on "three dots" and **License** the image
8. **View details** again and assert that the image is Used in Page

The image is still used in Page, so Used in should display that the image is Used in page
